internal/core/src/index/VectorMemIndex.cpp

```diff
@@ -760,14 +761,9 @@ void VectorMemIndex<T>::LoadFromFile(const Config& config) {
                                "lost index slice data");
                     auto data = batch_data[file_name];
                     auto start_write_file = std::chrono::system_clock::now();
-                    auto written = file.Write(data->Data(), data->Size());
+                    file.Write(data->Data(), data->Size());
                     write_disk_duration_sum +=
                         (std::chrono::system_clock::now() - start_write_file);
-                    AssertInfo(
```

Comment:
Why ignore this exception here?

Reply:
File instance in `File.h` is already handing the exception.
So exception handling can be ignored here.